app/routes.py

```diff
@@ -97,9 +97,7 @@ def create_shortlink():
         })
     )
 
-    logger.info(
-        "Shortlink Creation Successful.", extra={"response": json.loads(response.get_data())}
-    )
+    logger.info("Shortlink Creation Successful.", extra={"json": json.loads(response.get_data())})
```

Comment:
Instead of `json.loads(response.get_data())`  simply uses `response.get_json()`. To be consistent with the other logs the response data should be in `response.json`; `extra={"response": {"json": response.get_json()}}` (see log in @after_request)